wof/core.py

```diff
@@ -49,7 +49,8 @@
 except ImportError:
     version = 'dev'
 
-curr_folder = os.path.abspath(os.path.dirname(__file__))
+_root_dir = os.path.abspath(os.path.dirname(__file__))
+_TEMPLATES = os.path.join(_root_dir, 'app', 'templates')
```

Comment:
I am investigating that, and more of the extra "fat" I am finding, and I will send a PR soon.